lib/factory-helper/timestamps.rb

```diff
@@ -0,0 +1,31 @@
+module FactoryHelper
+  class Timestamps < Base
+
+    class << self
+      def between(from= ::Time.at(0), to= ::Time.local(2050, 12, 31,  23,  23,  59, 999_999.999))
+        random_time(from, to)
+      end
+
+      def forward(from= ::Time.at(0), seconds= 60)
+        to = from + seconds
+        random_time(from, to)
+      end
+
+      def backward(from= ::Time.at(0), seconds= 60)
+        to = from - seconds
+        random_time(to, from)
+      end
+
+    private
+
+      def random_time(from, to)
+        diff_time = to - from
+        dif_in_nsec = (diff_time * 1_000_000_000).to_i
+        rand_in_nsec = FactoryHelper::Config.random.rand(dif_in_nsec-1) + 1 # avoids 0
+        rand_in_sec = rand_in_nsec.to_r / 1_000_000_000
+        from + rand_in_sec
+      end
```

Comment:
Why not
```ruby
def random_time_simple(from, to)
  time_diff = to - from
  rand_time = FactoryHelper::Config.random.rand(time_diff - 1) + 1
  from + rand_time
end

5.times do
  puts random_time_simple(Time.now - 1, Time.now + 1).iso8601(7)
end

# => 2015-11-22T00:42:24.0341640+02:00
# => 2015-11-22T00:42:23.7815417+02:00
# => 2015-11-22T00:42:23.9767937+02:00
# => 2015-11-22T00:42:24.1995736+02:00
# => 2015-11-22T00:42:24.2099311+02:00
```
This seems to address the original problem of `Time.between` returning whole-second timestamps.

Also, the exclusion of the two boundaries (`from` & `to`) via the `-1` and `+1` in the `random_time` method appears to be inconsistent with other gem methods that return a random value from a range. e.g. FH::Time & FH::Date do not do this.